ovos_tts_plugin_server/__init__.py

```diff
@@ -57,12 +60,9 @@ def get_tts(
         if not voice or voice == "default":
             params.pop("voice")
         if self.host:
-            if isinstance(self.host, str):
-                servers: List[str] = [self.host]
-            else:
-                servers = self.host
+            servers = self.host
         else:
-            servers = self.public_servers
+            servers = random.shuffle(self.public_servers)
```

Comment:
This `else` statement will never be reached because `self.host` always returns a list of strings now